code/modules/mob/mob_defines.dm

```diff
@@ -170,6 +170,7 @@
 
 	var/datum/dna/dna = null//Carbon
 	var/radiation = 0.0//Carbon
+	var/rad_tick = 0.0//Carbon
```

Comment:
Oh yeah, I use it in rejuvenate() as that resets radiation to 0 along with rad_tick